self-hosted/sentry.conf.py

```diff
@@ -163,7 +163,7 @@
 # numbers of the same events being sent to the API in a short amount of time.
 # (read: if you send any kind of real data to Sentry, you should enable buffers)
 
-SENTRY_BUFFER = "sentry.buffer.redis.RedisBuffer"
+SENTRY_BUFFER = "sentry.services.buffer.redis.RedisBuffer"
```

Comment:
This will require self-hosted users to modify their config files when they are upgrading their self-hosted instances. Otherwise, it will break their setups

https://github.com/getsentry/self-hosted/blob/43d4c5df0e036ffa417ce2087fe37abbe20a09e7/sentry/sentry.conf.example.py#L178